osu.Game/Screens/Play/HUD/MultiplayerGameplayLeaderboard.cs

```diff
@@ -175,8 +178,6 @@ private void usersChanged(object sender, NotifyCollectionChangedEventArgs e)
 
             trackedData.Frames.Add(new TimedFrame(bundle.Frames.First().Time, bundle.Header));
             trackedData.UpdateScore();
-
-            updateTotals();
```

Comment:
Why was this call removed? Seems unintended, and the corresponding commit message ("Fix MultiplayerGameplayLeaderboard not immediately updating totals on scoring mode change") does not justify this change in my view.

I haven't gone to check because it's a hassle (need at least 3 client instances) but I'm like 99% sure this is going to break team score updates in spectator.

Reply:
What's the specific case you're thinking of? The new placement of this call handles any scores changes, so unless there's some other display which is required (or there's special handling when a user leaves?) I'm not immediately sure which case you are referring to.

Reply:
Ok I think I majorly misread this. Somehow missed the for loop around the new placement of this call. Never mind 😅